website/docs/docs/cloud/about-cloud/browsers.md

```diff
@@ -22,3 +22,8 @@ You may still be able to access and use dbt Cloud even without using the latest
 To improve your experience using dbt Cloud, we suggest that you turn off ad blockers.
 :::
 
+### Browser sessions
+
+A session is a period of time during which you’re signed in to a dbt Cloud account from a browser. dbt Cloud will not automatically log you out after a set time. However, if you close your browser, this action ends your dbt Cloud session and logs you out. You'll need to re-login the next time you try to access dbt Cloud. 
```

Comment:
```suggestion
A session is a period of time during which you’re signed in to a dbt Cloud account from a browser. dbt Cloud will not automatically log you out after a set time. However, if you close your browser, it will end your session and log you out. You'll need to log in again the next time you try to access dbt Cloud. 
```